src/components/item-grid/index.js

```diff
@@ -9,19 +9,15 @@ const getSubtitle = (text, minPrice, maxPrice, t) => {
     if (minPrice || maxPrice) {
         return (
             <div className="item-group-subtitle-wrapper">
-                <div>
-                    <span>{`${formatPrice(minPrice)} - ${formatPrice(
-                        maxPrice,
-                    )}`}</span>
-                    <div className="note">{t('per slot')}</div>
-                </div>
+                <span>{`${formatPrice(minPrice)} - ${formatPrice(maxPrice)}`}</span>
+                <div className="note">{t('per slot')}</div>
             </div>
         );
     }
 
     return (
         <div className="item-group-subtitle-wrapper">
-            <div>{text}</div>
+            {text}
```

Comment:
```suggestion
            <p>{text}</p>
```

Would this be appropriate rather than just leaving free text?

Reply:
The text will be in a div wrapper, an empty div does nothing, but a <p> will create a paragraph and we don't have CSS rules for it so it may break formatting